lib/game_stats.rb

```diff
@@ -8,34 +8,33 @@ class GameStats
 
   def initialize(file_path, object)
     @games = csv_data(file_path, object)
-    # this smells
-    @team_stats = TeamStats.new("./data/teams.csv", Team)
   end
 
   def percentage_ties
     ties = @games.count do |game|
       game.away_goals == game.home_goals
     end
-    (100 * ties.fdiv(@games.length)).round(2)
+    ties.fdiv(@games.length).round(2)
   end
 
   def count_of_games_by_season
+    #sort by number/games_by_season
     @games.reduce(Hash.new(0)) do |games_by_season, game|
-      games_by_season[game.season] += 1
+      games_by_season[game.season.to_s] += 1
```

Comment:
Instead of changing season to string here, we could just not call `.to_i ` in the Game class.